cmd/util/env.go

```diff
@@ -17,25 +17,14 @@ limitations under the License.
 package util
 
 import (
-	"os"
 	"strconv"
 
 	"go.uber.org/zap"
 )
 
-// GetRequiredEnvOrFatal tries to get the value of an environment variable.
-// If the value is empty, it logs an error and calls os.Exit(1).
-func GetRequiredEnvOrFatal(key string, logger *zap.SugaredLogger) string {
-	value := os.Getenv(key)
-	if value == "" {
-		logger.Fatalf("No %v provided", key)
-	}
-	logger.Infof("%v=%v", key, value)
-	return value
-}
-
-func MustParseIntEnvOrFatal(key string, logger *zap.SugaredLogger) int {
-	value := GetRequiredEnvOrFatal(key, logger)
+// MustParseIntEnvOrFatal tries to parse string to int value.
+// if conversion fails it logs an error and calls os.Exit(1).
+func MustParseIntEnvOrFatal(key, value string, logger *zap.SugaredLogger) int {
```

Comment:
We should be able to ditch this with the changes proposed above.

Reply:
Removed completely